### DISCUSSION TITLE
How do I set up the parser config for my tests?

### DISCUSSION THREAD

**Discussion by mantou132** [swc-project/swc#9536] [Q&A] [answered]
I'm a newbie, I tried all afternoon but couldn't find the answer 😢 , thx

my test:

```rs
test_inline!(
    Default::default(),
    |_| as_folder(TransformVisitor),
    desc,
    r#"
        class A {
            @a a(){}
        }
    "#,
    r#"
        class A {
            @a a(){}
        }
    "#
);
```

Unexpected token `@`. Expected identifier, string literal, numeric literal or [ for the computed key

**Comment by kdy1** [comment, marked as answer]
`Default::default()` is actually `Syntax::Es(EsSyntax::default())`
